.buildkite/kind-in-docker.yml

```diff
@@ -34,3 +34,23 @@
     - kubectl cluster-info
     - kubectl get nodes
     - kubectl get pods --all-namespaces
+
+    # Install Helm
+    - curl -Lo helm.tar.gz https://get.helm.sh/helm-v3.12.2-linux-amd64.tar.gz
+    - tar -zxvf helm.tar.gz
+    - mv linux-amd64/helm /usr/local/bin/helm
+    - helm repo add kuberay https://ray-project.github.io/kuberay-helm/
+    - helm repo update
+
+    # Install KubeRay operator
+    - pushd ray-operator
+    - IMG=kuberay/operator:nightly make docker-build
```

Comment:
Or `make docker-image`?

Reply:
Sure, I guess that will skip the unit tests and make the test run faster.  Any other reason? Let me go ahead and make that change

Reply:
> I guess that will skip the unit tests and make the test run faster.

yes